readme.md

```diff
@@ -8,8 +8,23 @@ See the "docs" directory for documentation and EIPs, and the "sharding" director
 
 ## Installation
 ### Environment
+
 Please refer to [pyethereum - Developer-Notes](https://github.com/ethereum/pyethereum/wiki/Developer-Notes)
 
+##### macOS
+
+  1. Switch to supported version of Python (i.e. using [virtualenv](https://github.com/pypa/virtualenv) or [pyenv](https://github.com/pyenv/pyenv))
+
+    ```shell
+    pyenv install 3.5.0; pyenv global 3.5.0; pyenv versions;
```

Comment:
Could you help us to change to the version more than or equaled to `3.6.0`?

Reply:
Yes, I'm using PyEnv and when I run `pyenv install --list` it shows that the latest version of 3.6 is `3.6.4rc1`, but it also shows `3.7.0a3` since Python 3.7 Alpha has been released. I'll see if we can incorporate Python 3.7.0a3. Reference: What's new in Python 3.7 https://docs.python.org/3.7/whatsnew/3.7.html